src/index.cts

```diff
@@ -134,8 +134,14 @@ export type ClientOpts = {
   clientKey?: string
 }
 
+export type JsPlaintext =
+  | string
+  | number
+  | Record<string, unknown>
+  | JsPlaintext[]
```

Comment:
It looks like the type here is recursive (as in nested arrays of `JsPlaintext` are allowed). Is that intended? If yes, can we please add tests for the a recursive type. If not, can we please refactor the type to not be recursive?

Reply:
Got it. I think it'd be worth adding a test to `./integration-tests` to cover that things line up as expected between TS and Rust for a nested array.